nuve/nuveAPI/resource/tokensResource.js

```diff
@@ -164,7 +164,9 @@ exports.create = (req, res) => {
       }
       log.info(`message: createToken success, roomId: ${currentRoom._id}, ` +
         `serviceId: ${currentService._id}`);
-      res.send(tokenS);
+      if (!res.headersSent) {
```

Comment:
why do we need to do this check here? by looking at the code I didn't find any case where we could have already sent the response.

Reply:
i think resean is some callback call twice, see #1435

Reply:
it is easy to reappear when stress testing. i had fixed it